Error while saving scenarios

Serializing scenarios to JSON (For example, while saving it using the editor), results in the following exception trace:
```
InfrastructureManager.Modules.Scenario.Exception.Output.ScenarioIOException: Scenario example.scenario could not be saved
        at InfrastructureManager.Modules.Scenario.ScenarioEditor.scenarioToFile(ScenarioEditor.java:93)
        at InfrastructureManager.Modules.Scenario.ScenarioEditor.execute(ScenarioEditor.java:51)
        at InfrastructureManager.ModuleManagement.PlatformModule.lambda$new$0(PlatformModule.java:33)
        at InfrastructureManager.ModuleManagement.Runner.run(Runner.java:53)
        at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class InfrastructureManager.Configuration.CommandSet and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) (through reference chain: InfrastructureManager.Modules.Scenario.Scenario["runner"]->InfrastructureManager.ModuleManagement.Runner["connections"]->java.util.ArrayList[0]->InfrastructureManager.ModuleManagement.Connection["commands"])
        at com.fasterxml.jackson.databind.exc.InvalidDefinitionException.from(InvalidDefinitionException.java:77)
        at com.fasterxml.jackson.databind.SerializerProvider.reportBadDefinition(SerializerProvider.java:1276)
        at com.fasterxml.jackson.databind.DatabindContext.reportBadDefinition(DatabindContext.java:400)
        at com.fasterxml.jackson.databind.ser.impl.UnknownSerializer.failForEmpty(UnknownSerializer.java:71)
        at com.fasterxml.jackson.databind.ser.impl.UnknownSerializer.serialize(UnknownSerializer.java:33)
        at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728)
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:770)
Input >
        at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178)
        at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serializeContents(IndexedListSerializer.java:119)
        at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serialize(IndexedListSerializer.java:79)
        at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serialize(IndexedListSerializer.java:18)
        at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728)
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:770)
        at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178)
        at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728)
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:770)
        at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178)
        at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480)
        at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319)
        at com.fasterxml.jackson.databind.ObjectMapper._writeValueAndClose(ObjectMapper.java:4485)
        at com.fasterxml.jackson.databind.ObjectMapper.writeValue(ObjectMapper.java:3682)
        at InfrastructureManager.Modules.Scenario.ScenarioEditor.scenarioToFile(ScenarioEditor.java:91)
        ... 4 more
```

More over, the file generated is wrong in structure

--- a/src/main/java/InfrastructureManager/ModuleManagement/PlatformInput.java
+++ b/src/main/java/InfrastructureManager/ModuleManagement/PlatformInput.java
@@ -4,7 +4,6 @@ import InfrastructureManager.ModuleManagement.Exception.Execution.ModuleExecutio
 import InfrastructureManager.PlatformObject;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
-@JsonIgnoreProperties({"runner", "ownerModule"})
 public abstract class PlatformInput extends PlatformObject {
 
     private final String name;

--- a/src/main/java/InfrastructureManager/Modules/Scenario/Scenario.java
+++ b/src/main/java/InfrastructureManager/Modules/Scenario/Scenario.java
@@ -15,7 +15,8 @@ import java.util.concurrent.Semaphore;
 /**
  * Class representing an scenario, as an object with a name and a list of events
  */
-@JsonIgnoreProperties({"startBlock","startTime","current", "pausedTime", "resumedTime","started"})
+@JsonIgnoreProperties({"startBlock","startTime","currentIndex", "pausedTime",
+        "resumedTime","started", "name", "runner", "ownerModule"})
 public class Scenario extends PlatformInput {
 
     private final List<Event> eventList;

